### ISSUE TITLE
Kill Terminal isn't destroying all processes

### ISSUE THREAD

**Issue by urmel9** [microsoft/vscode#113812]
### Environment details
- VSCode Version: 1.51.x && 1.52.x
- OS Version: Windows 10

### Description
Hello and happy new year :),

my problem is that my nodejs server doesn't destory himself when I kill the terminal.
It worked for a long time, but nearly 1 or 2 months ago it's doesnt destroy anymore properly.

### Actual behavior
If I want to start my nodejs again, it says:
`[nodemon] app crashed - waiting for file changes before starting...`
It is because the nodeServer is already started and in taskmanager are still zombie-processes: _Node.js: Server-side javascript_

### Steps to Reproduce:
1. Start ur node server with terminal
2. Kill terminal
3. Try to start again or look into taskmanager to see not all processes got closed

Does this issue occur when all **extensions** are disabled?: Yes

 ### Workaround
A possible solution is that I enter in **Windows Terminal** to kill all node processes:
`Taskkill /IM node.exe /F`
![2021-01-04 Windows Terminal](https://user-images.githubusercontent.com/67114570/103640815-f9b07080-4f50-11eb-919d-89669f64b415.png)

### Expected behavior
If I kill vs-code terminal it should destroy all processes started with same terminal. (Like earlier)

### My Questions
I don't know if it's a problem of node or of vs-code. But I opened the issue, because it has worked earlier and not anymore now.
Does anybody have an idea and can help me with a longterm-solution? If u have any questions, feel free to ask.

Thank you :)

### References
There is a similar problem with .NET:
[ Node.js processes don't quit after stopping debugging](https://github.com/dotnet/aspnetcore/issues/5204)

**Comment by meganrogge** [comment]
Duplicate of #86472
